cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -302,6 +300,12 @@ public static OrderedProperties getOrderedProperties(String configFileName) {
                     properties.addStringPairs(is);
                 } catch (IOException e) {
                     log.warn("Unable to read default log config from '{}'", configFileName);
+                } finally {
```

Comment:
Previously `addStringPairs(is)` was responsible for closing the stream passed to it. If we're changing that, presumably there's code in `addStringPairs(is)` which can now be removed. But check that all callers of `addStringPairs(is)` are updated to reflect the new arrangement.